template/src/ontology/Makefile.jinja2

```diff
@@ -219,13 +219,19 @@ RELEASE_ASSETS = \
   $(REPORT_FILES){% endif %}
 
 .PHONY: all_assets
-all_assets: $(ASSETS)
+all_assets: $(ASSETS) {% if project.ensure_valid_rdfxml %}check_rdfxml_assets{% endif %}
 
 .PHONY: show_assets
 show_assets:
 	echo $(ASSETS)
 	du -sh $(ASSETS)
 
+check_rdfxml_%: %
```

Comment:
should probably also be declared as .PHONY?

Reply:
I have seen nothing in Make’s documentation suggesting this would work for pattern rules (and indeed it does not seem to work).

The only way to make it work would be to declare as `.PHONY` all the possible expansions of the rule, based on the known `MAIN_PRODUCTS`. Too cumbersome for the little benefits this would bring, in my opinion.